pkg/handlers/search_files.go

```diff
@@ -0,0 +1,73 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/artmoskvin/hide/pkg/model"
+	"github.com/artmoskvin/hide/pkg/project"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	queryKey = "query"
+)
+
+type SearchFilesHandler struct {
+	ProjectManager project.Manager
+}
+
+func (h SearchFilesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
Are we using `type` and `content` anywhere here?

Reply:
Let's discuss this. Depends how we want to do with file (name) search.